statics/js/components/inject-form.js

```diff
@@ -217,16 +217,19 @@ Vue.component('inject-form', {
     getIP: function(node) {
       md = node.metadata;
```

Comment:
if after: 

```
md = node.metadata;
```

you append:
```
md = md.Neutron ? md.Neutron : md;
```

and then the rest if the code is left un modified

Reply:
done.